### DISCUSSION TITLE
Holley DTZ541.ZDCA

### DISCUSSION THREAD

**Discussion by JKatGithub67** [arendst/Tasmota#20400] [Q&A]
Hi. I'd like to use a Tasmota WIFI Adapter with ESP8266EX chip to read SML data from a Holley Dtz54.ZDCA1 power meter with the script https://tasmota.github.io/docs/Smart-Meter-Interface/#holley-dtz541-zdba-sml. Unfortunately all values are 0 and the ServerID ist empty.
The Tasmota WIFI Adapter is new. How may I check the TX/RX data? Is there a console command? Is there a simple possibility to check, the IR-sensors and the SML-Version of the smart-meter is affected? Known issues with this smart-meter are a) weak IR-LED, b) error in the SML it speaks. (described here https://github.com/volkszaehler/libsml/issues/75) 
Thanks

**Comment by spueli99** [comment]
Hi, 

Have you rotated the reading head 180 degrees?

Check Hichi FAQ for mirror test. 

https://youtu.be/VuXpzKetOhc?si=ZMlut4eLqS_Fot3B

https://sites.google.com/view/hichi-lesekopf/faq
